libs/service-portal/sessions/src/components/LogTable/LogTableMobile.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC } from 'react'
+import { Box, Divider, Icon, Text } from '@island.is/island-ui/core'
+import Person from '../PersonIcon/PersonIcon'
+import * as styles from '../LogTable/LogTable.css'
+import { SessionsSession } from '@island.is/api/schema'
+import { formatNationalId, getSessionType } from '../../utils/utils'
+import { useAuth } from '@island.is/auth/react'
+import { SessionType } from '../../lib/types/sessionTypes'
+import { dateFormat, timeFormat } from '@island.is/shared/constants'
+import { useLocale } from '@island.is/localization'
+
+interface LogTableProps {
+  sessions: SessionsSession[]
+}
+
+const ExpandedDivider = () => (
+  <div className={styles.divider}>
+    <Divider />
+  </div>
+)
+
+const LogTableMobile: FC<LogTableProps> = ({ sessions }) => {
+  const { userInfo } = useAuth()
+  const { formatDateFns } = useLocale()
+
+  return (
+    <>
+      {sessions.map((session: SessionsSession, index: number) => {
+        const type = getSessionType(session, userInfo?.profile.nationalId ?? '')
+
+        return (
+          <div style={{ width: '100%' }} key={index}>
+            <ExpandedDivider />
+            <Box
+              paddingY={3}
+              display={'flex'}
+              flexDirection={'column'}
+              rowGap={'gutter'}
+            >
+              <Box
+                display="flex"
+                justifyContent="spaceBetween"
+                alignItems="center"
+              >
+                <Text as="h5" variant="h5">
+                  {session.client.name}
+                </Text>
+                <Box>
+                  <Text variant="small">
+                    {formatDateFns(session.timestamp, dateFormat.is)}
```

Comment:
Curious about localisation. Does `formatDateFns` do anything? Can we make it automatic?

Reply:
`formatDateFns` uses by default the format `dd MMM yyyy` and then uses the locale to localise the months.
But the format of the date in our design is `dd.MM.yyyy` and with the time as `HH:mm`.
The question is if we want to convert the date format to i.e. `dd/MM/yyyy` and time as `hh:mm aaa` when the user switches to EN.

We could update the `formatDateFns` to support two format strings and it could automatically the one matching the locale. But doesn't scale well if island.is does decide to add another locale.